charts/tfy-jspolicy-configs/templates/pod-volume-mounts.yaml

```diff
@@ -78,6 +78,12 @@ spec:
           print("Invalid volume mount details: " + JSON.stringify(mountDetails));
           exit();
         }
+        // skip if the volume mount already exists
+        if (container.volumeMounts.find(volumeMount => volumeMount.name === volumeName)) {
```

Comment:
Future TODO: allowing multiple volumeMounts from the same volume as long as mountPath and subPath are not same